lido_sdk/config.py

```diff
@@ -3,4 +3,4 @@
 MULTICALL_MAX_WORKERS: int = 6
 MULTICALL_MAX_RETRIES: int = 5
 MULTICALL_POOL_EXECUTOR_TIMEOUT: int = 30
-VALIDATE_POOL_EXECUTOR_TIMEOUT: int = 10
+VALIDATE_POOL_EXECUTOR_TIMEOUT: int = 100
```

Comment:
is it in ms? after switching to batches of 1000 keys, will the script have enough to validate the keys?

Reply:
so it's in seconds, then it's fine